keyboards/keychron/k6/keymaps/ansi/keymap.c

```diff
@@ -88,3 +95,24 @@ void keyboard_post_init_user(void) {
     // debug_keyboard=true;
     // debug_mouse=true;
 }
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case KC_MISSION_CONTROL:
+            if (record->event.pressed) {
+                host_consumer_send(0x29F);
+            } else {
+                host_consumer_send(0);
+            }
+            return false;  // Skip all further processing of this key
+        case KC_LAUNCHPAD:
+            if (record->event.pressed) {
+                host_consumer_send(0x2A0);
+            } else {
+                host_consumer_send(0);
+            }
+            return false;  // Skip all further processing of this key
+        default:
+            return true;  // Process all other keycodes normally
+    }
+}|
```

Comment:
why reinvent the wheel? see https://github.com/SonixQMK/qmk_firmware/blob/sn32_master/keyboards/keychron/k12/keymaps/ansi/keymap.c#L33-L35

`KC_LAUNCHPAD` is also available with a key combo

Reply:
two reasons: 

1. using the built-in keycodes is a better 1:1 with how the dedicated MC and LP keys (i.e. on Magic Keyboards) work on macOS
2. this is the same way that Keychron themselves handle MC and LP [in qmk main](https://github.com/qmk/qmk_firmware/blob/fe8b7619d30f9571da386b4eff9103a5d6ca48d4/keyboards/keychron/q2/rev_0111/keymaps/default/keymap.c#L75)

I saw that the other K-series boards had done it the way you mentioned, but it felt hacky overall to use a key combo. those can be remapped in the system; personally, mine isn't even mapped to F3